.github/requirements/pip-requirements-macOS

```diff
@@ -0,0 +1,21 @@
+boto3==1.19.12
+hypothesis==6.56.4
+expecttest==0.1.3
+librosa>=0.6.2
+mpmath==1.2.1
+networkx==2.8.7
+numba==0.56.0; platform_machine == "arm64"
+numba<=0.49.1; platform_machine == "x86_64"
+opt-einsum>=3.3
+psutil==5.9.1
+pynvml==11.4.1
+pygments==2.12.0
+pytest==7.2.0
+pytest-xdist==3.0.2
+pytest-rerunfailures==10.2
+pytest-shard==0.1.2
+scipy==1.9.0; platform_machine == "arm64"
+scipy==1.6.3; platform_machine == "x86_64"
```

Comment:
Perhaps as followup: why can't we run both on 1.9.0?

Reply:
This is copied from the shell script, so let me try if this is also work with 1.9.0